app/views/shared/_cart_items.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="row event-page-ticket-listing event-page-inner">
+  <table class="table table-hover event-tickets">
+    <thead>
+
+      <tr>
+        <th></th>
+        <th></th>
+        <th>Title</th>
+        <th>Description</th>
+        <th>Location</th>
+        <th>Section</th>
+        <th>Row</th>
+        <th>Seat</th>
+        <th>Delivery Method</th>
+        <th colspan=2>Price</th>
+      </tr>
+    </thead>
+    <tbody>
+        <% @items.each do |item| %>
+        <tr class="event-row">
+          <td>
+            <div class="rotate event-category">
+              <p><%= item.event.category_name %></p>
+            </div>
+          </td>
+          <td>
+            <div class="event-date-group">
+              <p class="event-month"><%= item.event.month %></p>
+              <p class="event-date"><%= item.event.day_of_month %></p>
+              <p class="event-day"><%= item.event.day_of_week %></p>
```

Comment:
good use of the model methods! It's so easy to see now what it'll output! :star: